src/translations/en.global.json

```diff
@@ -11,15 +11,15 @@
     "account.available.error": "User not found.",
 
     "actions": {
-       "CreateAccount": "New account created: ${receiverId}",
-       "DeleteAccount": "Account deleted: ${receiverId}",
-       "DeployContract": "Contract deployed: ${receiverId}",
-       "FunctionCall": "Called method ${methodName} in contract @${receiverId}",
-       "Transfer": "Transferred ${deposit} to ${receiverId}",
+       "CreateAccount": "New account created: @${receiverId}",
+       "DeleteAccount": "Account deleted: @${receiverId}",
+       "DeployContract": "Contract deployed: @${receiverId}",
+       "FunctionCall": "Called method: ${methodName} in contract: @${receiverId}",
```

Comment:
maybe it makes sense to use dot notation?

`Called method ${receiverId}.${methodName}`